dev/test-dependencies.sh

```diff
@@ -126,8 +121,8 @@ for HADOOP_HIVE_PROFILE in "${HADOOP_HIVE_PROFILES[@]}"; do
   dep_diff="$(
     git diff \
     --no-index \
-    dev/deps/spark-deps-$HADOOP_HIVE_PROFILE \
-    dev/pr-deps/spark-deps-$HADOOP_HIVE_PROFILE \
+    dev/deps/spark-deps-$HADOOP_HIVE_PROFILE-scala-$SCALA_BINARY_VERSION \
```

Comment:
I guess you need to rename the existing manifest files and to add new manifest files for Scala 2.13, @slothspot .

Reply:
@dongjoon-hyun  Shall I interpret old files as scala-2.12 before rename or do some content-based test to derive which profile been used to obtain one?
And to clarify usage of this script: shall it iterate over both scala versions and generate two manifests or rely upon profile selected by the `./dev/change-scala-version`?